shap/plots/colors/_colorconv.py

```diff
@@ -725,7 +725,7 @@ def _scale(a, n, m, copy=True):
             dtype = "uint{}".format(mnew)
         n = int(np.ceil(n / 2) * 2)
         warn(
-            "Downcasting {} to {} without scaling because max " "value {} fits in {}".format(
+            "Downcasting {} to {} without scaling because max value {} fits in {}".format(
```

Comment:
NIT: I would prefer an f-string here but we can merge it in as is too.

Reply:
Since colorconv is vendored code (it's not written by us), I'll prefer to leave it in as is for now. Thanks